internal/service/privatelinkendpointservice/resource_privatelink_endpoint_service_test.go

```diff
@@ -17,6 +18,37 @@ func TestAccNetworkRSPrivateLinkEndpointServiceAWS_Complete(t *testing.T) {
 	resource.Test(t, *testCase)
 }
 
+func TestAccNetworkRSPrivateLinkEndpointServiceAWS_Failed(t *testing.T) {
+	acc.SkipTestForCI(t) // needs AWS configuration
```

Comment:
nit: Is it possible this test can be enabled in the CI?

Reply:
Yes, thanks for pointing out AWS provider was not needed! Removed CI skip 👍

Reply:
Test now runs successfully in CI

Reply:
great that it runs in CI 👏